spec/integration/spec.yml

```diff
@@ -66,3 +66,9 @@ spec:
     name: elasticsearch
     required: false
     $ref: "./elasticsearch/spec.yml"
+  - description: Errors configuration file
+    type: file
+    contentMediaType: "application/x-yaml"
+    name: "errors.yml"
```

Comment:
I don't like the name "errors" so much, as this is going to end up in built packages. Would it be an option something like `lint.conf/yaml`? Or `validation.yml`?

Reply:
I don't find any other better names. I opted for `validation.yml`